packages/rule-no-p3p/README.md

```diff
@@ -0,0 +1,25 @@
+# no-p3p (`no-p3p`)
+
+Don't use P3P related headers
+
+## Why is this important?
+
+P3P -- Platform for Privacy Preferences Project -- is an outdated technology
+that is meant to allow browsers to programmatically check privacy policies. In
+case it's not kept in check with the actual privacy policies, it's a legal risk.
```

Comment:
Can you please add a link to the spec, and sources for the "outdated technology" and  "legal risks"?